selendroid-standalone/src/main/java/io/selendroid/standalone/server/handler/ProxyToDeviceHandler.java

```diff
@@ -83,6 +83,9 @@ public Response handleRequest(HttpRequest request, JSONObject payload) throws JS
     while (retries-- > 0) {
       try {
         response = proxyRequestToDevice(request, session, url, method);
+        if (response == null) { // Unknown command
+          return new SelendroidResponse(sessionId, StatusCode.UNKNOWN_COMMAND);
```

Comment:
shouldn't we be sending back the 404 to the client too? and not just wrapping the 404 from the server?